CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,111 @@
+# Contributor Covenant Code of Conduct
+
+Please note that this project is released with a Contributor Code of Conduct. By participating in this project you agree to abide by its terms.
+
```

Comment:
Yeah, we might as well. Though, the more we change it, the harder it'll be to update if the contributor covenant updates again.